moveit_core/robot_model/include/moveit/robot_model/joint_model.h

```diff
@@ -42,9 +42,14 @@
 #include <vector>
 #include <map>
 #include <iostream>
-#include <moveit_msgs/JointLimits.h>
+// #include <moveit_msgs/JointLimits.h>
+#include <moveit_msgs/msg/joint_limits.hpp>
 #include <random_numbers/random_numbers.h>
 #include <Eigen/Geometry>
+#include <rcutils/logging_macros.h>
+
+#define ROS_DEBUG_NAMED RCUTILS_LOG_ERROR
+#define ROS_WARN_NAMED RCUTILS_LOG_ERROR
```

Comment:
i think these defines should be moved to a new file called ``moveit_core/macros/include/moveit/macros/logging_macros.h`` with the ``#include <rcutils/logging_macros.h>`` include also

Reply:
We have already created a file for this task https://github.com/ros-planning/moveit2/blob/master/moveit_core/logging/include/moveit/logging/logging.h.

Reply:
I very much believe this should be part of the rcutils header. Has this been discussed there?
Unless, of course, the ROS2 maintainers really want to emphasize breaking every single patch that could be compatible between ROS1 and ROS2.

Reply:
@v4hn you're welcome to take that up there, that would be cleaner but more of a political debate!

Reply:
See: https://github.com/ros-planning/moveit2/pull/7#discussion_r258702364

Reply:
Pushed to https://github.com/ros-planning/moveit2/issues/21

Reply:
I'd rather we not change anything in logging than have a half solution here